.ado/pipelines/templates/stages-full-release.yaml

```diff
@@ -456,6 +479,9 @@ stages:
 
               - template: steps-set-pipeline-variables.yaml # load set-pipeline-variables function
 
+              # TODO - Delete Azure Load Test resource (if still present)
```

Comment:
There's a possibility that in some situations the load test resources remain - for example when the e2e pipeline is executed (with destroy unchecked) and later the e2e pipeline is executed again on the same branch with destroy selected but w/o the load test selected.

Reply:
What do you suggest? Global?

Reply:
yeah probably, given that it targets AFD. or the first stamp?

Reply:
Problem is that the issue could affects both - locust and azure load test - we'd then need to modify both to deploy their resources (in case of locust ACI + KV + Storage and in case of MALT only the MALT service) to an existing RG and for standalone into a dedicated resource group.

Reply:
Added `&> /dev/null` to not fail when these resource groups do not exist.

Reply:
Ok, had to improve this a bit more to work.

https://github.com/Azure/Mission-Critical-Online/blob/df209a66786f1dba7da53f397fd71874455a553b/.ado/pipelines/templates/stages-full-release.yaml#L482-L508